src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesApiServiceTest.kt

```diff
@@ -1,11 +1,15 @@
+@file:OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
not sure why you need these but you do, e.g not sure why a test helper is "experimental"